lib/glimr_api_client/register_new_case.rb

```diff
@@ -0,0 +1,56 @@
+module GlimrApiClient
+  class RegisterNewCase < Base
+    def initialize(params)
+      @params = params
+    end
+
+    private
+
+    def check_request!
+      errors = []
+      [
+        :jurisdictionId,
+        :onlineMappingCode
+      ].each do |required|
+        errors << required if request_body.fetch(required).blank?
+      end
+      raise RequestError, errors unless errors.blank?
+    end
+
+    def endpoint
+      '/registernewcase'
+    end
+
+    def request_body
```

Comment:
unk...

Sorry, not helpful...Mainly just meant "wow, that's a lot of body params..."